luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -98,7 +110,8 @@ public SearchProvider getSearchProvider(String domain) {
     /**
      * Bare minimum.
```

Comment:
Not strictly related to this PR, but we should not be relying on
`default` to select the `MapStore`.  We should explicitly look for the MapStore
class name, and throw an exception under `default`. Otherwise, a typo could
mean customers end up using an in-memory KeyValueStore instead of whatever
store they intended, and that could be very bad if it's a sufficiently large
dimension.

Also, I thought we were going to complete KeyValueStore before moving onto
tables. This function doesn't look particularly done, in that it still has
`TODO: Magic Values` and we haven't implemented a factory for the RedisStore
yet.

Reply:
The factory for RedisStore is scheduled to be pushed until we are finished. That being said, I agree with doing it _while not over-clogging this PR_. 

I can open a quick PR to implement KeyValueStore before opening the ones for LogicalTable.

Reply:
That's fine.